### PR TITLE
Container endpoint

### PR DESCRIPTION
**Pull request by MarkGibbons**
### Description

Document the container endpoint and methods.  

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by MarkGibbons** [comment]
I'm torn about this one.  I can't image using these endpoints and Chef may not want them documented.    /containers doesn't show up in the side index of the API documentation. I think that's because we need a blank line inserted.  If we leave the doc as is that part still needs to happen.  Cheers.

File: content/api_chef_server.md [hunk 1]
```diff
@@ -2604,6 +2606,184 @@ The response is similar to:
 </tbody>
 </table>
 
+### POST
+
+The `POST` method is used to create a container.
+
```
Comment (markan) [inline content/api_chef_server.md:2710]: Add here something like `NOTE: while this method is implemented, it is not supported, and use will likely break your system, or not have the effect you expect.`

File: content/api_chef_server.md [hunk 2]
```diff
@@ -2604,6 +2606,184 @@ The response is similar to:
...
+/containers/NAME
+----------------
+
+The `/containers/Name` endpoint has the following methods: `DELETE`, `GET`.
+
+### DELETE
+
+The `DELETE` method is used to remove a container.
+
```
Comment (markan) [inline content/api_chef_server.md:2778]: Add here something like `NOTE: while this method is implemented, it is not supported, and use will likely break your system, or not have the effect you expect.`

**Review by markan — CHANGES_REQUESTED** [review]
Thank you for this PR. You rightly identify the API as somewhat problematic. 

There's certainly nothing secret about these APIs. However they are functionally useless, and possibly dangerous (DELETEing a container probably will break your system). They were implemented for completeness, and the routes for the ACLs naturally were built off of them.

I still think it's worthwhile documenting these. I would suggest adding a note for the POST/DELETE methods. Something like "NOTE: while this method is implemented, it is not supported, and use will likely break your system" for each of those methods.

File: content/api_chef_server.md (hunk 2 above)
Comment (MarkGibbons) [reply content/api_chef_server.md:2778]: In the queue. Thanks

File: content/api_chef_server.md [hunk 3]
```diff
@@ -2657,6 +2661,10 @@ The `/containers/Name` endpoint has the following methods: `DELETE`, `GET`.
 
 The `DELETE` method is used to remove a container.
 
+The `/containers/Name` endpoint has the following methods: `DELETE`, `GET`.
+
+Note: Using the `DELETE` method of the `/containers` endpoint may have unexpected effects and is likely to break your systemr. Use of this method is not supported.
```
Comment (markan) [inline content/api_chef_server.md]:
```suggestion
Note: Using the `DELETE` method of the `/containers` endpoint may have unexpected effects and is likely to break your system. Use of this method is not supported. 
```

**Review by markan — APPROVED** [review]

**Comment by markan** [comment]
Thanks again for this!

**Review by kagarmoe — APPROVED** [review]
tenor-42074873

**Pull request merged**